packages/tools/create-kadena-app/templates/angular/tsconfig.app.json

```
@@ -0,0 +1,14 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./out-tsc/app",
+    "types": [".kadena/pactjs-generated"]
+  },
```

This is extended from `./tsconfig.json` so not necessary

```suggestion
    "outDir": "./out-tsc/app",
  },
```